packages/login/src/components/StyledLoginForm.js

```diff
@@ -26,15 +28,36 @@ export const StyledLoginFormInputWrapper = styled.div`
   }
 `
 
-export const StyledLoginFormWrapper = styled.div`
+export const StyledLoginButton = styled(Button)`
+  &&&& {
```

Comment:
&&&& seems a bit odd to me. are you sure there is nothing a bit more elegant to solve this :)